api/app/templates/webpack/readme.md

```diff
@@ -0,0 +1,3 @@
+# Webpack Templates Folder
+
+Front-end production template builds are copied here by webpack. Files in this folder should not be committed to git.
```

Comment:
Couldn't we just add a `.gitignore` to this directory then and ignore everything?